components/service/sync-logins/src/test/java/mozilla/components/service/sync/logins/AsyncLoginsStorageTest.kt

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.sync.logins
+
+import kotlinx.coroutines.async
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.plus
+import mozilla.appservices.logins.DatabaseLoginsStorage
+import mozilla.appservices.logins.LoginsStorage
+import mozilla.appservices.logins.MemoryLoginsStorage
+import mozilla.components.concept.storage.SyncError
+import mozilla.components.concept.storage.SyncOk
+import mozilla.components.concept.storage.SyncStatus
+import mozilla.components.concept.storage.SyncableStore
+
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+import kotlinx.coroutines.newSingleThreadContext
+import kotlinx.coroutines.runBlocking
+import java.io.File
+
+@RunWith(RobolectricTestRunner::class)
+class AsyncLoginsStorageTest {
+    private val testMainScope = CoroutineScope(newSingleThreadContext("Test"))
+
+    @Test
+    fun `can do some stuff`() {
```

Comment:
A bit more concise:
```
fun `can do some stuff`() = runBlocking {
...
}
```

Reply:
I needed to put the `runBlocking` _inside_ the `use { }` block.  I would be happy to be proven wrong.